### PR TITLE
Don't break gitpod.yml with template string

### PR DESCRIPTION
**Pull request by mashehu**

### CODE REVIEW

File: nf_core/pipeline-template/.gitpod.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ tasks:
     command: |
       pre-commit install --install-hooks
       nextflow self-update
-
+{% if code_linters %}
```
Comment (mirpedrol) [inline nf_core/pipeline-template/.gitpod.yml:7]: I think this one we can keep, to remove the new line
```suggestion
{%- if code_linters %}
```

File: nf_core/pipeline-template/.gitpod.yml [hunk 2]
```diff
@@ -3,9 +3,8 @@ tasks:
   - name: Update Nextflow and setup pre-commit
     command: |
       pre-commit install --install-hooks
-      nextflow self-update
+      nextflow self-update {%- if code_linters %}
```
Comment (mirpedrol) [inline nf_core/pipeline-template/.gitpod.yml:6]: not needed here
```suggestion
      nextflow self-update {% if code_linters %}
```

File: nf_core/pipeline-template/.gitpod.yml [hunk 3]
```diff
@@ -3,9 +3,8 @@ tasks:
...
 
 vscode:
   extensions:
-    #{%- if code_linters -%}
-    - nf-core.nf-core-extensionpack # https://github.com/nf-core/vscode-extensionpack{% endif %}
+    - nf-core.nf-core-extensionpack {%- endif -%}
```
Comment (mirpedrol) [inline nf_core/pipeline-template/.gitpod.yml:10]: also not needed
```suggestion
    - nf-core.nf-core-extensionpack {% endif %}
```

Comment (mirpedrol) [reply nf_core/pipeline-template/.gitpod.yml:10]: don't we want to keep the comment with the url?

Comment (mashehu) [reply nf_core/pipeline-template/.gitpod.yml:10]: ah, sorry, yes

**Review by mirpedrol — APPROVED** [review]

**Pull request merged**